frontend/src/interfaces/IJobData.ts

```diff
@@ -0,0 +1,70 @@
+// import { ContractStatus } from '../enums/ContractStatus';
+import { BigNumber } from 'ethers';
+
+export enum IJobState {
```

Comment:
I would just break this file up a bit more.

Reply:
Made #67 to remind us to do this.